webvh/webvh/did/models/operations.py

```diff
@@ -100,10 +100,52 @@ class WebvhCreateWitnessInvitationSchema(OpenAPISchema):
 class WebvhUpdateSchema(OpenAPISchema):
     """Request model for updating a Webvh DID."""
 
-    # class UpdateStateSchema(OpenAPISchema):
-    #     "Webvh DID state update schema."""
-
-    class UpdateParametersSchema(OpenAPISchema):
+    class DocumentUpdateSchema(OpenAPISchema):
+        """Webvh DID document update schema."""
+
+    # class DocumentOperationsSchema(OpenAPISchema):
```

Comment:
I'd rather not have this committed to main. Maybe you can just keep it locally if you want to save it.